pkg/api/api.go

```diff
@@ -61,11 +62,17 @@ type Config struct {
 	// initialized, the custom config handler will be used instead of
 	// DefaultConfigHandler.
 	CustomConfigHandler ConfigHandler `yaml:"-"`
+
+	// These allow and are used to configure the addition of HTTP Header fields to logs
+	LogHTTPRequestHeaders   bool                `yaml:"log_http_request_headers"`
```

Comment:
Do we need this boolean here?
Is not enough to just see if the headers is not emptu?